packages/redot-stringify/test.js

```diff
@@ -0,0 +1,16 @@
+import {test} from 'node:test'
+import * as assert from 'node:assert/strict'
+import {unified} from 'unified'
+import redotStringify from './index.js'
```

Comment:
```suggestion
import redotStringify from 'redot-stringify'
```

With export maps, you cal also use the package *itself* in tests. Making them closer to what users have to do!